queue/src/fiskaltrust.Middleware.Storage.SQLite/Migrations/012_ftJournalFRCopyPayload.sqlite3

```diff
@@ -4,9 +4,9 @@
     CashBoxIdentification     NVARCHAR(2147483647) NOT NULL,
     Siret                     NVARCHAR(2147483647) NOT NULL,
     ReceiptId                 NVARCHAR(2147483647) NOT NULL,
-    ReceiptMoment             DATETIME NOT NULL,
+    ReceiptMoment             BIGINT NOT NULL,
```

Comment:
TL;DR There is no DATETIME type in SQLite.

This still worked somehow for net6 which is why it was not noticed. In net461 it failed though.

It is not possible in SQLite to change the type of a column so we'd need to create a new column and keep the old one.

Since the 1.3 FR Queue is not in use yet we decided to change the migration instead. New queues will have the correct datatype and old queues are not FR queues and don't use this table.